src/_igraph/convert.c

```diff
@@ -2684,6 +2684,73 @@ int igraphmodule_PyObject_to_vector_int_ptr_t(PyObject* list, igraph_vector_ptr_
   return 0;
 }
 
+/**
+ * \ingroup python_interface_conversion
+ * \brief Converts a Python list of lists to an \c igraph_vector_list_t
+ *        (containing \c igraph_vector_int_t items).
+ *
+ * \param o the Python object representing the list of lists
+ * \param m the address of an uninitialized \c igraph_vector_list_t
+ * \return 0 if everything was OK, 1 otherwise. Sets appropriate exceptions.
+ */
+int igraphmodule_PyObject_to_vector_list_t(PyObject* list, igraph_vector_list_t* vec) {
```

Comment:
It's much better to call `vector_list_t` objects a "list" so they are never confused with vectors. `list` is a better name than `vec`.

Reply:
agreed, I called the list `veclist` and the vector `vec`